Mathlib/Algebra/Homology/HomotopyCategory/MappingCone.lean

```diff
@@ -541,6 +541,93 @@ lemma lift_desc_f {K L : CochainComplex C ℤ} (α : Cocycle K F 1) (β : Cochai
     liftCochain_v_descCochain_v φ α.1 β α' (Cochain.ofHom β') (zero_add 1) (neg_add_self 1) 0
     (add_zero 0) n n n (add_zero n) (add_zero n) n' hnn', Cochain.ofHom_v]
 
+
+section
+
+open Preadditive Category
+
+variable (H : C ⥤ D) [H.Additive]
+  [HasHomotopyCofiber ((H.mapHomologicalComplex (ComplexShape.up ℤ)).map φ)]
+
+/-- If `H : C ⥤ D` is an additive functor and `φ` is a morphism of cochain complexes
+in `C`, this is the comparison isomorphism (in each degree) between the image
+by `H` of `mappingCone φ` and the mapping cone of the image by `H` of `φ`. -/
+@[simps]
+noncomputable def mapHomologicalComplexXIso' (n m : ℤ) (hnm : n + 1 = m) :
+    ((H.mapHomologicalComplex (ComplexShape.up ℤ)).obj (mappingCone φ)).X n ≅
+      (mappingCone ((H.mapHomologicalComplex (ComplexShape.up ℤ)).map φ)).X n where
+  hom := H.map ((fst φ).1.v n m (by omega)) ≫
+      (inl ((H.mapHomologicalComplex (ComplexShape.up ℤ)).map φ)).v m n (by omega) +
+      H.map ((snd φ).v n n (add_zero n)) ≫
+        (inr ((H.mapHomologicalComplex (ComplexShape.up ℤ)).map φ)).f n
+  inv := (fst ((H.mapHomologicalComplex (ComplexShape.up ℤ)).map φ)).1.v n m (by omega) ≫
+      H.map ((inl φ).v m n (by linarith)) +
```

Comment:
Not at a computer so I can't check: could this be `omega`? No problem if there is a good reason.